lib/puppet/provider/cisco_tacacs_server/cisco.rb

```diff
@@ -114,6 +114,15 @@ def destroy
     @tacacs_server = nil
   end # destroy
 
+  def encryption_password
```

Comment:
BTW, for cisco providers we can take care of this in the type.  We only need the special getter code in the netdev providers.